src/pages/CreateAccount.js

```diff
@@ -26,19 +26,23 @@ export default function CreateAccount({ api, signInAndRedirect }) {
 
   const [errorMessage, setErrorMessage] = useState("");
 
-  let history = useHistory();
   let inviteCodeInputDOM = useRef();
 
   useEffect(() => {
     api.getSystemLanguages((languages) => {
-      languages.learnable_languages.sort((a, b) => (a.name > b.name) ? 1 : -1)
-      languages.native_languages.sort((a, b) => (a.name > b.name) ? 1 : -1)
+      languages.learnable_languages.sort((a, b) => (a.name > b.name ? 1 : -1));
+      languages.native_languages.sort((a, b) => (a.name > b.name ? 1 : -1));
       setSystemLanguages(languages);
       inviteCodeInputDOM.current.focus();
     });
     // eslint-disable-next-line
   }, []);
 
+  /* always sends the user to a next onboarding step */
+  function handleRedirect(linkToRedirect) {
```

Comment:
this function is not needed. it simply is a synonim for redirect. and it's actually a bit more confusing because there's another `handleRedirect` in the project that does something else. now, that one I propose to refactor, but still -- this code would be clear if you simply called `redirect()` below.

Reply:
✅ Done